src/Apps/ViewingRoom/Routes/Statement/Components/ViewingRoomStatements.tsx

```diff
@@ -0,0 +1,63 @@
+import React from "react"
+import { Box, Sans, Spacer, Serif } from "@artsy/palette"
+
+export const ViewingRoomStatements: React.FC = props => {
+  return (
+    <Box>
+      {/* Pull quote */}
+      <Box>
+        <Sans size="8" textAlign="center">
+          Traces of bodies abound in Kwak’s hollowed forms, burnished
+          ectoplasmic impressions asserting a queer immanence. Their poses
+          simultaneously suggest supplication, pleasure, distress, nostalgia,
+          the past, the present, and the future.
+        </Sans>
+      </Box>
+
+      <Spacer my={4} />
+
+      {/* Statement 1 */}
+      <Box>
+        <Serif size="5">
+          Young Joon Kwak (b. 1984 in Queens, NY) is a LA-based
+          multi-disciplinary artist working primarily through sculpture,
+          performance, video, and collaboration. Kwak's work reimagines the
+          form, functionality, and materiality of objects in order to propose
+          different ways of viewing and interpreting bodies and the physical and
+          social spaces that govern our everyday lives as mutable, inclusive,
+          and open-ended. Kwak's work reimagines the form, functionality, and
+          materiality of objects in order to propose different ways of viewing
+          and interpreting bodies and the physical and social spaces that govern
+          our everyday lives as mutable, inclusive, and open-ended.
+        </Serif>
+      </Box>
+
+      <Spacer my={4} />
```

Comment:
I do still think we should implement [a Stack](https://chakra-ui.com/stack) — I find this whole spacer pattern a lil weird. But maybe it's just unfamiliar. (Though I don't like all the extra DOM)

Reply:
Yeah, I used something like that [here](https://github.com/damassi/reaction/blob/27418dec416d0fe6125801609c786c9affa849e0/src/Apps/ViewingRoom/Routes/Statement/index.tsx#L15) but I think we adding Stack would improve our API a bit.